src/core/cdrom.cc

```diff
@@ -1567,6 +1570,17 @@ class CDRomImpl : public PCSX::CDRom {
         StopCdda();
         lidSeekInterrupt();
     }
+
+    void logCDROM (int command) {
+        if (command >= 0x100) return; // a hack to fight the hacks in this file... yikes
```

Comment:
We may want to instead have a `bool delayed = command & 0x100` and add a "[DELAYED]" keyword in the output. Also `switch (command & 0xff)`

Reply:
Ohhhhhhhhh I really like this idea actually